tensorflow/core/framework/tensor.cc

```diff
@@ -58,6 +59,7 @@ limitations under the License.
 #include "tensorflow/core/platform/protobuf.h"
 #include "tensorflow/core/platform/tensor_coding.h"
 #include "tensorflow/core/platform/types.h"
+#include "tensorflow/tsl/util/tensor_bundle/byte_swap_array.h"
```

Comment:
Sorry I didn't notice in your last comment, but these should be directly under `util`, and not `tensor_bundle` anymore.  Otherwise we're going to run into the same layering issue as before.

Reply:
Got it. I'll move `byte_swap_array.h` and `byte_swap_array.cc` to `tsl/util` folder.